CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,12 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
```

Comment:
CC or Apache? Just this file or others too?

Reply:
oh even here this is not needed. removed